ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/AstProcessorApplication.java

```diff
@@ -0,0 +1,23 @@
+package edu.usb.argos.ASTProcessor;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import java.util.List;
+
+@SpringBootApplication
+public class AstProcessorApplication {
```

Comment:
I think that this class initialize application and that should not contain logic. The verification of the `JavaParser` library, I think it is better to do it only from the unit tests.

Reply:
It was corrected